covasim/webapp/index.html

```diff
@@ -116,6 +116,18 @@ <h4 class="my-2">Campaign Builder</h4>
             {{sim_length.tip}}
           </b-tooltip>
         </div>
+        <div class="form-group">
+          <label for="country">Demographic:</label>
```

Comment:
@cliffckerr, not sure if we should move this to the population parameters or if it is important enough to expose in the general options

Reply:
@sayers24 There should just be a "location" drop-down that sets everything automatically

Reply:
FYI, I'm working on the other PRs that will impact this, and see also https://github.com/InstituteforDiseaseModeling/covasim/pull/171 for more merge hell :)

Reply:
Would we want to call out that the location drop down is more than just a preset? The model uses this to take age distribution into account, correct? We could add a little more descriptive text to the tooltip.